tests/test_add_okf_frontmatter.py

```
@@ -0,0 +1,200 @@
+    """Test case for add_okf_frontmatter.py functions."""
+
+    def setUp(self):
+        self._orig_cwd = os.getcwd()
+        self._tmp_dir = tempfile.TemporaryDirectory()
+        os.chdir(self._tmp_dir.name)
+
+    def tearDown(self):
+        os.chdir(self._orig_cwd)
+        self._tmp_dir.cleanup()
+
+    def _write(self, filepath, content):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_write`

Add return type annotation: `None`

(ANN202)